gcp/rakefiles/vars.rb

```diff
@@ -52,7 +55,10 @@ def self.set_vars(env, project_type)
       ENV["BILLING_ID"] = "01A0E1-B0B31F-349F4F"  # RtF Billing Account
     end
 
-    # Hack to avoid changes in gpii-version-updater
+    set_versions
```

Comment:
This method shouldn't run during unit tests for `set_secrets`.

Can you remove it from here and add it to `:set_vars` instead?

Reply:
Done